src/main/scala/is/hail/annotations/aggregators/RegionValueProductAggregator.scala

```diff
@@ -0,0 +1,72 @@
+package is.hail.annotations.aggregators
+
+import is.hail.annotations._
+
+class RegionValueProductBooleanAggregator extends RegionValueAggregator {
```

Comment:
Was there consensus on whether to leave in code for `RegionValueSumBooleanAggregator` and `RegionValueProductBooleanAggregator` even though there's no Python path to them?